_usage/limitations-of-babelfish.md

```diff
@@ -302,21 +304,77 @@ features are discovered, and additional features are added to MS SQL:
 | `WAITFOR/RECEIVE` | This syntax is not supported. |
 | `WAITFOR TIME` | This syntax is not supported. |
 | `WITH ENCRYPTION` clause | This syntax is not supported for functions, procedures, triggers, or views. |
-| `CREATE/ALTER/DROP WORKLOAD CLASSIFIER` | Functionality related to this syntax is not supported |
+| `CREATE/ALTER/DROP WORKLOAD CLASSIFIER` | Functionality related to this syntax is not supported. |
 | `CREATE/DROP WORKLOAD CLASSIFIER` | This syntax is not supported. |
 | `WRITETEXT` | This syntax is not supported. |
 | `OPENXML()` | This function is not supported. |
-| XML datatype with schema (xmlschema) | XML type without schema is supported |
-| XML methods | This includes .VALUES(), .NODES(), and other methods |
+| XML datatype with schema (xmlschema) | XML type without schema is supported. |
+| XML methods | This includes .VALUES(), .NODES(), and other methods. |
 | XML indexes | XML indexes are not supported. |
 | XPATH expressions | This syntax is not supported. |
 | WITH XMLNAMESPACES construct | This syntax is not supported. |
 | `WITHOUT SCHEMABINDING` clause | Not supported in functions, procedures, triggers, or views. The object will be created, but as if `WITH SCHEMABINDING` was specified. |
 | `CREATE/ALTER/DROP SELECTIVE XML INDEX` clause | This syntax is not supported. |
 | `CREATE/ALTER/DROP XML SCHEMA COLLECTION` | This syntax is not supported. |
 
-However, more is missing than just features. There are also some
-corner cases which will provide the end user with [different
-behavior](/docs/usage/missing-features) than otherwise expected from MS SQL. It is
-important to understand those corner cases, as well.
+However, more is different than just features.
+For example, backup and recovery work differently in SQL Server and PostgreSQL,
+and you will have to use the PostgreSQL tools to backup Babelfish.
+It is important to understand such differences as well.
 
+## Dealing with limitations
+
+In this section, we've written a lot about limitations, unsupported features and
+such. However, how do you deal with these things in real life? If Babelfish
+runs a SQL statement that doesn't return a result set that is identical to the
+result set when run against SQL Server, the SQL statement will fail.
```

Comment:
^^^^
this suggests some sort of run-time check against a result set, but that is not how it works.

Reply:
That is pre-existing text, I just copied it over from `_usage/missing-features.md`, which was deleted.
The wording is from 14bb8bd313b211d0d55e30afc128069f8656db57 by @xaviersierra; perhaps he can comment on that?

Reply:
I am the principal database engineer on the babelfish project, and I designed the escape hatches. Please use my change suggestions.

Reply:
Tell that to @xaviersierra, not me.

Reply:
This content was provided by CyberTec, it can be found [here](https://github.com/cybertec-postgresql/babelfish_doc/blob/102b2f6e1a59622f7f8ac6f54243b666697f8f54/pages/mydoc/babelfish_query_handling.md). I've just added to the website

Reply:
This documentation was provided by CyberTec, and can be found [here](https://github.com/cybertec-postgresql/babelfish_doc/blob/102b2f6e1a59622f7f8ac6f54243b666697f8f54/pages/mydoc/missing_features.md?plain=1#L497). I simply added it to the website

Reply:
I see. Since I have not the faintest clue what an escape hatch is, please provide a description that I can use.

Reply:
Ok, I think I have addressed that by simply removing the sentence. The behavior is described two sentences down anyway.

Reply:
I think I see the light now. 0cfa834bc1b27d59015a48a9773d76dbd76f67a8 should improve this whole section.